While executing „zigbee things“   An unexpected error occurred during execution.

Using OH 4.3.3

While executing zigbee things i get an error:

```
openhab> zigbee things                                                                                                                                 
An unexpected error occurred during execution.
```

The log shows

```

2025-03-01 12:13:11.020 [ERROR] [b.core.io.console.ConsoleInterpreter] - An error occurred while executing the console command.
java.lang.NullPointerException: Cannot invoke "com.zsmartsystems.zigbee.ZigBeeNode.getLinkQualityStatistics()" because "node" is null
	at org.openhab.binding.zigbee.console.internal.ZigBeeConsoleCommandExtension.handleThingsCommand(ZigBeeConsoleCommandExtension.java:158) ~[?:?]
	at org.openhab.binding.zigbee.console.internal.ZigBeeConsoleCommandExtension.handleCommand(ZigBeeConsoleCommandExtension.java:119) ~[?:?]
	at org.openhab.binding.zigbee.console.internal.ZigBeeConsoleCommandExtension.execute(ZigBeeConsoleCommandExtension.java:92) ~[?:?]
	at org.openhab.core.io.console.ConsoleInterpreter.execute(ConsoleInterpreter.java:57) [bundleFile:?]
	at org.openhab.core.io.console.karaf.internal.CommandWrapper.execute(CommandWrapper.java:78) [bundleFile:?]
	at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:68) [bundleFile:4.4.6]
	at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:86) [bundleFile:4.4.6]
	at org.apache.felix.gogo.runtime.Closure.executeCmd(Closure.java:599) [bundleFile:4.4.6]
	at org.apache.felix.gogo.runtime.Closure.executeStatement(Closure.java:526) [bundleFile:4.4.6]
	at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:415) [bundleFile:4.4.6]
	at org.apache.felix.gogo.runtime.Pipe.doCall(Pipe.java:416) [bundleFile:4.4.6]
	at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:229) [bundleFile:4.4.6]
	at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:59) [bundleFile:4.4.6]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:840) [?:?]

```

--- a/org.openhab.binding.zigbee.console/src/main/java/org/openhab/binding/zigbee/console/internal/ZigBeeConsoleCommandExtension.java
+++ b/org.openhab.binding.zigbee.console/src/main/java/org/openhab/binding/zigbee/console/internal/ZigBeeConsoleCommandExtension.java
@@ -155,16 +155,20 @@ public class ZigBeeConsoleCommandExtension extends AbstractConsoleCommandExtensi
             } else {
                 ZigBeeThingHandler handler = (ZigBeeThingHandler) thing.getHandler();
                 ZigBeeNode node = handler.getNode();
-                ZigBeeLinkQualityStatistics lqi = node.getLinkQualityStatistics();
+                if (node == null) {
+                    builder.append("** Node not found!");
+                } else {
+                    ZigBeeLinkQualityStatistics lqi = node.getLinkQualityStatistics();
 
-                builder.append(String.format("%-15s  ", node.getLogicalType()));
-                builder.append(String.format("%04X  ", node.getNetworkAddress()));
-                builder.append(String.format("%-16s  ", node.getIeeeAddress()));
-                builder.append(String.format("%-5s  ", handler.isDeviceInitialized() ? "TRUE" : "FALSE"));
-                builder.append(
-                        lqi.getLastReceivedLqi() == null ? "   " : String.format("%3d  ", lqi.getLastReceivedLqi()));
-                builder.append(
-                        lqi.getLastReceivedLqi() == null ? "   " : String.format("%3d  ", lqi.getLastReceivedRssi()));
+                    builder.append(String.format("%-15s  ", node.getLogicalType()));
+                    builder.append(String.format("%04X  ", node.getNetworkAddress()));
+                    builder.append(String.format("%-16s  ", node.getIeeeAddress()));
+                    builder.append(String.format("%-5s  ", handler.isDeviceInitialized() ? "TRUE" : "FALSE"));
+                    builder.append(lqi.getLastReceivedLqi() == null ? "   "
+                            : String.format("%3d  ", lqi.getLastReceivedLqi()));
+                    builder.append(lqi.getLastReceivedLqi() == null ? "   "
+                            : String.format("%3d  ", lqi.getLastReceivedRssi()));
+                }
             }
             builder.append("\n");
         }

